backend/api/helper/permissions.py

```diff
@@ -0,0 +1,16 @@
+from flask_security import current_user
+
+
+def current_user_has_permission_to_change(entry) -> bool:
```

Comment:
yes I'm aware of that, but I'd rather have an extra wrapper for is_permitted called cur_permitted or something then. I can definitely see use for the is_permitted method if we add more user management stuff for the admin panel, or requesting project edits from a permitted user